viewer/main.js

```diff
@@ -21,6 +23,20 @@ let tileset;
 let attenuation = true;
 let show_bboxes = false;
 
+const customShader = new Cesium.CustomShader({
+    LightingModel: Cesium.LightingModel.UNLIT,
+    vertexShaderText: `
+    void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput) {
+        vsOutput.pointSize = 4.0;
+    }
+    `,
+    fragmentShaderText: `
+    void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material) {
+        material.diffuse = vec3(1.0);
```

Comment:
Unfortunately custom shaders doesn't yet support metadata. But we can at least make use of the feature IDs to apply styling